x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/back_to_list_panel/back_to_list_panel.tsx

```diff
@@ -37,7 +29,7 @@ export const BackToListPanel: FC = () => {
             defaultMessage: 'Return to the analytics management page.',
           }
         )}
-        onClick={redirectToAnalyticsManagementPage}
+        href={analyticsManagementPageLink}
```

Comment:
As above, you should add text to the PR description to say what this is fixing as it doesn't seem to be related to the other changes in this PR/

Reply:
Updated here https://github.com/elastic/kibana/pull/84594/commits/b47459d369ba5485d798e7a976bed1b25c4f5c94